pkg/commands/git_commands/commit_loader.go

```diff
@@ -310,6 +310,17 @@ func (self *CommitLoader) getInteractiveRebasingCommits() ([]*models.Commit, err
 		return nil, nil
 	}
 
+	// See if the current commit couldn't be applied because it conflicted; if
+	// so, add a fake entry for it
+	if conflictedCommitSha := self.getConflictedCommit(todos); conflictedCommitSha != "" {
+		commits = slices.Prepend(commits, &models.Commit{
```

Comment:
Do we need to prepend? `commits` is an empty slice here so append should be fine

Reply:
Oh sure, see d8869655bb34.